docs/api/cluster/icdm.rst

```diff
@@ -5,26 +5,24 @@ Intercluster Distance Maps
 
 Intercluster distance maps display an embedding of the cluster centers in 2 dimensions with the distance to other centers preserved. E.g. the closer to centers are in the visualization, the closer they are in the original feature space. The clusters are sized according to a scoring metric. By default, they are sized by membership, e.g. the number of instances that belong to each center. This gives a sense of the relative importance of clusters. Note however, that because two clusters overlap in the 2D space, it does not imply that they overlap in the original feature space.
 
-.. code:: python
+.. plot::
```

Comment:
Previously, this example in the documentation had been broken out into two blocks&mdash;one to generate the dataset and one to `fit` the model and `poof` the visualizer. So in order to have it work with the new plot directive, I combined them into one and moved all of the import statements to the top of the block.